charts/opensearch/ci/ci-rbac-enabled-values.yaml

```diff
@@ -318,12 +320,21 @@ terminationGracePeriod: 120
 
 sysctlVmMaxMapCount: 262144
 
+startupProbe:
+  enabled: false
```

Comment:
This value doesn't seem to be enabled anywhere in the CI values? Also I think having `enabled` under this key will break the template because it just inserts the whole contents of `startupProbe`, expecting it to be valid probe YAML.

Reply:
The reason I added the flag was because of the api support for lower version cluster. Read [more](https://github.com/helm/helm/issues/7731#issuecomment-594993686). Somehow CI was not supporting the probe. . I will add a version check instead and remove the enabled key itself. How does that go in?

Reply:
@smlx Are we good to close this comment?